src/Event/EventTest/Suites.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DAL.Events;
+using EventApplication.Entities;
+using TestsCore;
+
+namespace EventTest
+{
+	internal class Suites
+	{
+		private static Dictionary<GetValuesSuites, List<Event>> suiteEventsMap => new()
+		{
+			{ GetValuesSuites.Empty, [] },
+			{ GetValuesSuites.OneValue, [new()] },
+			{ GetValuesSuites.ThreeValues, [new(),new(),new()] },
+			{ GetValuesSuites.ManyValues, [new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),] },
+		};
+
+		private static Dictionary<GetValuesSuites, List<SeatDetails>> suiteSeatsMap => new()
+		{
+			{ GetValuesSuites.Empty, [] },
+			{ GetValuesSuites.OneValue, [new()] },
+			{ GetValuesSuites.ThreeValues, [new(),new(),new()] },
+			{ GetValuesSuites.ManyValues, [new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),new(),] }
+		};
+
+		public static IList<Event> GetEvents(GetValuesSuites suite)
+		{
+			return suiteEventsMap[suite];
+		}
+
+		public static IList<SeatDetails> GetSeats(GetValuesSuites suite)
+		{
+			return suiteSeatsMap[suite];
+		}
```

Comment:
All this can be replaced with just: 
```
public List<T> GetCollection<T>(int size) where T: new() => Enumerable.Range(0, size).Select(index => new T()).ToList();
```
Then put this method in some extension or helper class in Common project and reuse it for all instances in all tests

Reply:
Named it `ListGenerator`, suites go extinct.